pyo3-ffi/src/lib.rs

```diff
@@ -65,7 +65,7 @@
 //!
 //! ```toml
 //! [build-dependency]
```

Comment:
Not from this PR, but I think worth correcting:

```suggestion
//! [build-dependencies]
```